### DISCUSSION TITLE
Course Sistem Rekomendasi

### DISCUSSION THREAD

**Discussion by naturallyfunny** [teamalgoritma/community#750] [Q&A] [answered]
Ada yang tau course indo atau non indo yang bagus terkait pembuatan dan pengembangan sistem rekomendasi?

Dengan R maupun Python

**Comment by Yusraf002** [comment, marked as answer]
Dear @ArdiantheGreat ,

Terimakasih atas pertanyaannya, untuk course terkait sistem rekomendasi Kak Ardian bisa coba baca artikel berikut
https://algotech.netlify.app/blog/skincare-recommender-system/ 

pada link berikut dijelaskan mengenai collaborative filtering menggunakan python.

Demikian yang dapat kami sampaikan, jika membantu boleh ditandai mark as answer. Terima kasih :)

**Reply by naturallyfunny** [reply]
Terima kasih!
